api/python/cellxgene_census/src/cellxgene_census/_get_anndata.py

```diff
@@ -58,12 +64,20 @@ def get_anndata(
             Columns to fetch for ``obs`` and ``var`` dataframes.
         obsm_layers:
             Additional obsm layers to read and return in the ``obsm`` slot.
+        add_obs_embeddings:
+            Additional embeddings to be returned as part of the ``obsm`` slot.
+            Use :func:`get_all_available_embeddings` to retrieve available embeddings
+            for this Census version and organism.
+        add_var_embeddings:
+            Additional embeddings to be returned as part of the ``varm`` slot.
+            Use :func:`get_all_available_embeddings` to retrieve available embeddings
+            for this Census version and organism.
```

Comment:
Naming seems off.  When you want to include/add/fetch an X or obsm layer, you use `X_layers` and `obsm_layers`, but for an embedding it is `add_obs_embeddings` (i.e., includes a verb).  Why not stick with the consistent schema, a la `obs_embeddings` and `var_embeddings`?  or alternatively `obs_contributed_embeddings`, ....?

Reply:
`obs_embeddings` seems fine. `obs_contributed_embeddings` could work but it might be confusing since it is also used for collaborations embeddings. Maybe it's worth adding a paragraph at the beginning with a quick explanation on what these embeddings are?

Reply:
`obs_embeddings` and `var_embeddings` are good names.